scripts/install_inspec_master.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+set -euo pipefail
+
+export HAB_NONINTERACTIVE=true
+export HAB_NOCOLORING=true
+export HAB_LICENSE="accept-no-persist"
+
+install_inspec() {
+  hab pkg install core/gcc --binlink --force
+  hab pkg install core/ruby --binlink --force
+
+  tmp_dir=$(mktemp -d)
+  git clone --depth=1 https://github.com/inspec/inspec "$tmp_dir"
+  cd "$tmp_dir"
+  bundle
+  bundle exec rake install
```

Comment:
This looks great. I have some scripts in inspec/inspec's expeditor dir to make it really easy to cache bundle artifacts. We can get that on another iteration. Cuts down ~3 minutes per. Maybe it's also possible to cache the hab pkgs?